platform/windows/detect.py

```diff
@@ -182,13 +182,14 @@ def get_opts():
 
     return [
         ("mingw_prefix", "MinGW prefix", mingw),
-        # Targeted Windows version: 7 (and later), minimum supported version
+        # Targeted Windows version: 10 (and later), minimum supported version
         # XP support dropped after EOL due to missing API for IPv6 and other issues
         # Vista support dropped after EOL due to GH-10243
+        # 7, 8 and 8.1 support dropped after EOL due to Direct3D 12 rendering driver addition
         (
             "target_win_version",
-            "Targeted Windows version, >= 0x0601 (Windows 7)",
-            "0x0601",
+            "Targeted Windows version, >= 0x0A00 (Windows 10)",
```

Comment:
The `>=` part is meant to show the min supported version, which doesn't seem to be Windows 10. IIUC we still support down to Windows 7 if passing `0x601`.

Reply:
Updated description to mention both.